src/networks/types/GOERLI.js

```diff
@@ -4,7 +4,7 @@ import { GOERLI } from '../tlds';
 import goerli from '@/assets/images/icons/network.svg';
 
 export default {
-  name: 'GöETH',
+  name: 'GOERLI',
```

Comment:
why did you change this to `GOERLI` ?

Reply:
There was an issue with matching the name as it is different than the type file name.  I don't remember where it was occurring, so I just reverted it.